contrib/scripts/FileCache.py

```diff
@@ -1,13 +1,16 @@
 import dataclasses
 import datetime
 import os
+import sys
 
-import AbstractFileFingerprint as aff
 import opencsp.common.lib.file.CsvInterface as ci
 import opencsp.common.lib.tool.time_date_tools as tdt
 import opencsp.common.lib.tool.file_tools as ft
 import opencsp.common.lib.tool.log_tools as lt
 
+sys.path.append('../../')
```

Comment:
Does this depend on having something special two directories up from the current directory? Are we sure this always be the case?

Reply:
I'm not sure what the best way to deal with this is.  I think it's either this or we add "path/to/contrib" to the PYTHONPATH env variable. thoughts?

Reply:
Perhaps we could define it relative to opencsp_code_dir()? This way, it doesn't matter the directory the user is running this from since opencsp_code_dir is based on the location of the opencsp folder itself. For example:
`sys.path.append(os.path.join(opencsp_code_dir(), '../contrib/scripts'))`

Reply:
That does seem better. I replaced all of these:
`sys.path.append('../../')`
with this:
`sys.path.append(os.path.join(orp.opencsp_code_dir(), '..'))`